gitarro.rb

```diff
@@ -8,9 +8,13 @@
 require_relative 'lib/gitarro/backend'
 
 b = Backend.new
-exit 0 if b.triggered_by_pr_number?
 
-prs = b.open_newer_prs
+prs = if b.options[:pr_number].nil?
```

Comment:
can we create an helper function for this?
It can resided in same file.

Or we can use guard clause for avoding if/else cognitive load.